ydb/core/tx/schemeshard/schemeshard_import__create.cpp

```diff
@@ -471,11 +471,12 @@ struct TSchemeShard::TImport::TTxProgress: public TSchemeShard::TXxport::TTxBase
         Y_ABORT_UNLESS(item.State == EState::BuildIndexes);
 
         const auto uid = MakeIndexBuildUid(importInfo, itemIdx);
-        if (!Self->IndexBuildsByUid.contains(uid)) {
+        const auto* infoPtr = Self->IndexBuildsByUid.FindPtr(uid);
+        if (!infoPtr) {
             return InvalidTxId;
         }
 
-        return TTxId(ui64(Self->IndexBuildsByUid.at(uid)->Id));
+        return TTxId(ui64((**infoPtr).Id));
```

Comment:
> Idiomatic variant would be (*infoPtr)->.

Maybe we have different understanding of Idiomatic ;)
But ok, I can change it for you.

> So infoPtr name is a bit inaccurate.

I want to insist on such naming because it's used in all other places with FindPtr too.
And something like `infoPtrPtr` will be confused in my opinion.